nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/spectra/GestureBindings.java

```diff
@@ -74,15 +75,17 @@ public void scroll(ScrollEvent event) {
         ChartBorder border = chart.hitBorder(x, y);
         double dx = event.getDeltaX();
         double dy = event.getDeltaY();
-        if (event.isControlDown() && border == ChartBorder.NONE) {
-            chart.scaleY(dy);
-        } else if (border == ChartBorder.RIGHT || border == ChartBorder.TOP) {
-            chart.updateProjectionScale(border, dy);
+        int scrollDirectionFactor = SystemUtils.IS_OS_MAC  ? 1 : -1;
```

Comment:
This assumes all Macs will use natural scrolling, and Windows and Linux will use traditional scrolling... both Mac and Linux can change the scroll direction in their settings...I couldn't find any way to detect this from java, if we want to add a setting for this, we could have one to switch our assumption of the scrollDirectionFactor within NMRFx but we couldn't specify if its natural of traditional scrolling direction...so didn't add a setting for this at this time